tasks/switch_static_services.yml

```diff
@@ -1,12 +1,12 @@
 ---
-# Patch app dest routes with src stack
+# Patch app dest static services with src stack
 
-- include_tasks: deploy_get_stamp_from_route.yml
+- include_tasks: deploy_get_stamp_from_static_service.yml
```

Comment:
actually, I don't understand why the name of this task starts by `deploy`. Why not call it `get_stamp_from_static_service`?

Reply:
Ok I get it, it's a prefix...